nuxt.config.js

```diff
@@ -132,5 +135,9 @@ export default {
     */
     extend (config, ctx) {
     }
+  }, 
+  publicRuntimeConfig: {
```

Comment:
I read up on this and it turns out that the `env` block in the config is used for things that are needed at _build_ time. I'm not sure when they added these runtime config features but I like it!